analyses/snv-callers/explore_variant_classifications/explore_nonsynfilter.Rmd

```diff
@@ -0,0 +1,307 @@
+---
+title: "Explore impact of Non Synonymous Filters"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL
+date: 2020
+---
+
+### Summary of Findings:
+
+This was an exploration of how TMB is affected by using nonsynonymous filters. 
+It addresses part 1 of [this OpenPBTA issue](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/729)
+As far as the TMB comparisons go, it doesn't matter if we use `Variant_Classification` filters or not. 
+They are highly correlated no matter what and the same general TCGA to PBTA differences seem to persist. 
+
+### Usage
+
+If both `run_caller_consensus_analysis-tcga.sh` and `run_caller_consensus_analysis-pbta.sh` have been run, you can run this command. 
+This will prepare the files needed for this notebook and run this notebook:
+
+```
+# bash run_explorations.sh
+```
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+- [Read in the TMB files](#read-in-the-tmb-files)
+  - [Read in PBTA data](#read-in-pbta-data)
+  - [Read in TCGA data](#read-in-tcga-data)
+- [Does the filter change a participant's TMB?](#does-the-filter-change-a-participants-tmb)
+  - [Plot PBTA data](#plot-pbta-data)
+  - [Plot TCGA data](#plot-tcga-data)
+- [Are some histologies affected more than others?](#are-some-histologies-affected-more-than-others)
+- [Does the filter affect the TCGA-PBTA comparison change?](#does-the-filter-affect-the-tcga-pbta-comparison-change)
+- [Plot the TMB plot with no filter data](#plot-the-tmb-plot-with-no-filter-data)
+- [Session Info](#session-info)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Setup
+
+```{r}
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+```
+
+```{r}
+source(file.path("..", "..", "tmb-compare-tcga", "util", "cdf-plot-function.R"))
+```
+
+```{r}
+dir.create("plots", showWarnings = FALSE)
+```
+
+## Read in the TMB files
+
+### Read in PBTA data
+
+Import the tmb calculations that are not filtered by nonsynonymous classifications.
+
+```{r}
+tmb_pbta_no_filter <- data.table::fread(file.path(
+  "..",
+  "results",
+  "no_filter",
+  "pbta-snv-mutation-tmb-coding.tsv"
+)) %>%
+  # This variable is weird when binding but we don't need it for the plot so we'll just remove it.
```

Comment:
tell me more?

Reply:
Hmm... It was weird when I used this kind of import in another notebook but now its fine, so I deleted this step.